src/modules/anim_controller.cpp

```diff
@@ -16,18 +16,20 @@
 #include "bluetooth/bluetooth_messages.h"
 #include "bluetooth/bluetooth_message_service.h"
 #include "leds.h"
+#include "drivers_nrf/scheduler.h"
 
 using namespace Animations;
 using namespace Modules;
 using namespace Config;
 using namespace DriversNRF;
 using namespace Bluetooth;
 
-#define MAX_ANIMS (20)
-#define TIMER2_RESOLUTION 33 //ms
+#define MAX_ANIMS 20
 
 namespace Modules::AnimController
 {
+	DelegateArray<AnimControllerClientMethod, 1> clients;
```

Comment:
Use `static` qualifier to make this symbol private to this file (not accessible from other compilation units)